controllers/commands/forward_command.js

```diff
@@ -28,19 +29,21 @@ async function forWardCommand(argv) {
             return null
         })
         await autoSwitchUser(url)
+
         const args = process.argv.slice(2);
         let command = args.join(' ')
         runCommandWithGit(command)
     }
 }
 
+
 const autoSwitchUser = async (url) => {
     if(!Boolean(url)) return false
     let repo = getRepo({url})
     if(repo == REPO_DOES_NOT_EXISTS) return false
     let user = getUserInfoById(repo.userID)
     if(user == USER_DOES_NOT_EXISTS) return false
-    if(!user.isDefault) await switchUser(user.username)
+    if(!user.isDefault) await switchUser({username: user.username})
```

Comment:
The argv of switchUser function is not an object.

![Screen Shot 2022-05-14 at 06 27 25](https://user-images.githubusercontent.com/53461381/168401070-a4679b26-e8f5-4959-932a-b1defce9563e.png)